src/queries/disputes.js

```diff
@@ -0,0 +1,107 @@
+import gql from 'graphql-tag'
+
+export const AllDisputes = gql`
+  subscription {
+    disputes(orderBy: createdAt, orderDirection: desc) {
+      id
+      createTermId
+      possibleRulings
+      finalRuling
+      lastRoundId
+      state
+      metadata
+      createdAt
+      subject {
+        id
+      }
+      rounds {
+        state
+        number
+        draftTermId
+        jurorsNumber
+        settledPenalties
+        jurorFees
+        delayedTerms
+        selectedJurors
+        coherentJurors
+        collectedTokens
+        createdAt
+        jurors {
+          juror {
+            id
+          }
+        }
+        appeal {
+          id
+          maker
+          appealedRuling
+          taker
+          opposedRuling
+          settled
+          createdAt
+        }
+      }
+    }
+  }
+`
+
+export const JurorDrafts = gql`
+  query JurorDrafts($id: ID!) {
+    juror(id: $id) {
+      id
+      drafts {
+        id
+        weight
+        rewarded
+        commitment
+        outcome
+        leaker
+        createdAt
+        round {
```

Comment:
@facuspagnuolo now that i am looking it again i think this is wrong because once i get the drafts from the juror i am doing this
 `result.data.juror.drafts.map(draft =>
          reduceDispute(draft.round.dispute))`
An i think that one dispute could be present in more than one draft of the same juror right? so that will end in duplicated disputes on "My disputes" tab for the juror right?

Reply:
yea, a juror could be drafted for the same dispute more than once in case they are selected to vote on different rounds

Reply:
is there a way to query that directly? probably i need to query  `disputes` passing the juror and check if in the last round the juror is present, not sure how to do that with graphql haha